src/definition/users/IUserContext.ts

```diff
@@ -0,0 +1,6 @@
+import { IUser } from './IUser';
+
+export interface IUserContext {
+    user: IUser;
+    performedBy?: IUser;
+}
```

Comment:
It would be good to add some comments here explaining app devs what is the purpose of the interface and the properties